makeservices/easywp

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Creates symlink to user's webroot. If the symlink exists and files
+# are present in webroot, they are backed up in the user's home directory.
+# A user's database is created or, if it already exists, the password 
+# is reset. WordPress is installed and the wp-config file is filled
+# out with the requisite information. Further configuration can be 
+# done by the user from a browser.  
+#
+# WARNING: This script should generally be run on a account without
+#          an existing website or database!
```

Comment:
I would be a little more explicit and summarize the potential danger: files get backed up, but the database password changes and the new WP install might try to migrate/clobber any existing database.

Reply:
If we expect users to run this, definitely make this a prompt before running. If only staff, it might be ok to leave it as a comment, but it still wouldn't hurt.

Reply:
From talking with @Baisang, I thought that the user would click a button on the OCF website (https://www.ocf.berkeley.edu/account/commands/) to set up WordPress and it would run this script on tsunami as that user. Then they setup WordPress without having to touch a terminal. If that's the case there should be a warning prompt, but on the web interface. If not then I agree there should be a cli warning. That said, we could pipe 'y' into the script if it was going to be called in response to a button click.

Reply:
I'm honestly not sure if anyon ever visits the commands page anyways. I also think it's not a good idea to put this script up there. It can do all sorts of bad things if used improperly, and, if the script fails, whoever ran the command will have no recourse available from the website.

This script is probably best left for staff to run in the lab after setting up a new group account, or for somewhat experienced users who know how to SSH and can help themselves or get email help if things break. We don't want to encourage people to shoot themselves in the foot without knowing it.

Reply:
I agree with mattmcal. Additionally, I feel that we should eventually start to phase out "commands". It's a bizarre concept (run specific commands over SSH over HTTP)  when most of these thing should be first-class features.

Commands like `paper` are almost embarrassing -- we should just show the user's quota when they're logged in on the site. The same could be said for `makemysql` and `quota`. `makehttp` isn't even necessary anymore unless you mess things up, since we run it automatically for new accounts.

Reply:
I agree the commands section is not great, but just thought it would be a place to put something like this theoretically so users could do the WordPress install without touching a terminal. So if there were some tool that would let users automagically install WordPress for themselves, where would a better place be for it? Something like ocfweb just calling to ocflib?

Reply:
For now, I agree that it makes more sense to just have this run by staff or something. So the prompt Matt suggested would be a good idea.

Reply:
If we were to do something like that, and I'm not saying we should, it could indeed be a celery task called by ocfweb. However, I don't really like the sound of having a celery task that just shells out to a bash script, nor do I like the sound of rewriting this script in python, since it depends on `wp-cli` and `make{http,mysql}`, so I'm not even sure how this hypothetically code would be organized.